src/plugins/intel_gna/src/gna_graph_compiler.cpp

```diff
@@ -663,6 +663,99 @@ void GNAGraphCompiler::finalizeConvolution1DPrimitive(InferenceEngine::CNNLayerP
     }
 }
 
+struct Convolution2DWeightsAdaptorData {
+    size_t original_kernel_width;
+    size_t original_kernel_height;
+    size_t effective_kernel_width;
+    size_t effective_kernel_height;
+    size_t input_channels;
+    size_t convolution_precision;
+    size_t kernels_num;
+    uint8_t* orginal_kernel_data;
+};
+
+class Convolution2DWeightsAdaptor {
```

Comment:
Pls provide unit tests for class Convolution2DWeightsAdaptor

Reply:
code will be removed